build/generate_bundle.sh

```diff
@@ -42,6 +45,7 @@ build_bundle_instructions() {
 }
 
 
+${OPERATOR_SDK} version
```

Comment:
This was definitely for debugging, and helped eventually to figure out that there were multiple operator-sdk versions needed